cmd/bosun/sched/notify.go

```diff
@@ -121,7 +125,11 @@ func (s *Schedule) sendNotifications(silenced SilenceTester) {
 				}
 				s.pendingUnknowns[n] = append(s.pendingUnknowns[n], st)
 			} else if silenced {
-				slog.Infoln("silencing", ak)
+				slog.Infof("silencing %s", ak)
+				continue
+			} else if !alert.Log && (!st.Open || !st.NeedAck) {
+				slog.Infof("Cannot notify acked or closed alert %s", ak)
```

Comment:
Can we log more info about the alert / notification here so we might be able to find when they are in the queue when they shouldn't be?